iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/ShuntCompensatorXmlTest.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.xml;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.ShuntCompensator;
+import com.powsybl.iidm.network.test.ShuntTestCaseFactory;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class ShuntCompensatorXmlTest extends AbstractXmlConverterTest {
+
+    @Test
+    public void test() throws IOException {
+        Network network = ShuntTestCaseFactory.create();
+        ShuntCompensator sc = network.getShuntCompensator("SHUNT");
+        sc.setProperty("test", "test");
+        roundTripXmlTest(network,
+                NetworkXml::writeAndValidate,
+                NetworkXml::read,
+                getVersionDir(IidmXmlConstants.CURRENT_IIDM_XML_VERSION) + "/shuntRoundTripRef.xml");
+    }
+}
```

Comment:
Shouldn't there be more tests here, to check an exception is thrown In case you try to import a shunt in IIDM 1_0 with regulations?

Reply:
Good point, I will add it